### PR TITLE
Add Jaeger projects for 2024 Term-1

### PR DESCRIPTION
**Pull request by yurishkuro**
Upstream issue https://github.com/jaegertracing/jaeger/issues/5084

### CODE REVIEW

**Comment by tanayvaswani** [comment]
@yurishkuro Are there any plans for React.js related ideas in this term?

File: programs/lfx-mentorship/2024/01-Mar-May/project_ideas.md [hunk 1]
```diff
@@ -18,4 +18,33 @@
...
 
+### Jaeger
+
+#### Jaeger-V2 Storage Backends
+
+- Description: Jaeger is a distributed tracing platform. Jaeger V2 is a major new version where we rebase all Jaeger backend components (agent, collector, ingester, and query) on top of the OpenTelemetry Collector. Currently only memory storage is wired in v2, we need to add Elasticsearch, Opensearch, Cassandra, Badger.
+- Expected Outcome: Build out full support in jaeger-v2 for all storage backends supported by jaeger-v1
+- Recommended Skills: Go, scripting, CI/CD
+- Mentor(s):
+  - Yuri Shkuro (@yurishkuro, github@ysh.us)
```
Comment (aliok) [inline programs/lfx-mentorship/2024/01-Mar-May/project_ideas.md:29]: Could you please add a back up mentor? This year, we really encourage having 2 mentors in all programs.

File: programs/lfx-mentorship/2024/01-Mar-May/project_ideas.md [hunk 2]
```diff
@@ -18,4 +18,33 @@
...
+- Upstream Issue: https://github.com/jaegertracing/jaeger/issues/5084
+
+#### Jaeger-V2 Observability
+
+- Description: Jaeger is a distributed tracing platform. Jaeger V2 is a major new version where we rebase all Jaeger backend components (agent, collector, ingester, and query) on top of the OpenTelemetry Collector. Currently jaeger-v2 components are initialized without observability clients. We need to instantiate appropriate logging, tracing, and metrics clients and pass them to the components. The existing code uses internal metrics API, which needs to be bridged to OTEL metrics to minimize code changes.
+- Expected Outcome: Achieve parity in observability of jaeger-v2 compared to jaeger-v1
+- Recommended Skills: Go, scripting, CI/CD
+- Mentor(s):
+  - Yuri Shkuro (@yurishkuro, github@ysh.us)
```
Comment (aliok) [inline programs/lfx-mentorship/2024/01-Mar-May/project_ideas.md:38]: Same here

File: programs/lfx-mentorship/2024/01-Mar-May/project_ideas.md [hunk 3]
```diff
@@ -18,4 +18,33 @@
...
+- Upstream Issue: https://github.com/jaegertracing/jaeger/issues/5084
+
+#### Jaeger-V2 Adaptive Sampling
+
+- Description: Jaeger is a distributed tracing platform. Jaeger V2 is a major new version where we rebase all Jaeger backend components (agent, collector, ingester, and query) on top of the OpenTelemetry Collector. Jaeger-v1 collector can serve sampling configuration to SDKs, and allows either static configuration (with hot reload) or adaptive sampling that continuously re-calculates the desired sampling probabilities. We need to enable all these capabilities in jaeger-v2.
+- Expected Outcome: Support adaptive sampling in jaeger-v2
+- Recommended Skills: Go, scripting, CI/CD
+- Mentor(s):
+  - Yuri Shkuro (@yurishkuro, github@ysh.us)
```
Comment (aliok) [inline programs/lfx-mentorship/2024/01-Mar-May/project_ideas.md:47]: Same here

**Review by aliok — COMMENTED** [review]
Sorry for asking changes that hasn't been documented yet, but here's the relevant PR for us (mentoring wg chairs) that we're working on: https://github.com/cncf/mentoring/pull/1088

**Comment by yurishkuro** [comment]
> Are there any plans for React.js related ideas in this term?

@tanayvaswani we have limited mentor bandwidth, so we are prioritizing jaeger-v2 over UI work.

File: programs/lfx-mentorship/2024/01-Mar-May/project_ideas.md (hunk 1 above)
Comment (yurishkuro) [reply programs/lfx-mentorship/2024/01-Mar-May/project_ideas.md:29]: added

**Review by nate-double-u — APPROVED** [review]
Thanks for this @yurishkuro & @jkowall!

Also, this term we’re doing an information session for mentors. We’d like to invite all the mentors participating to attend one of these if you're able:
* North America: [Jan 29, 2024 10:30 AM (Pacific Time)](https://zoom-lfx.platform.linuxfoundation.org/meeting/99504894798?password=54a6ec2f-d649-432b-a918-acd17720f163)
* EMEA + APAC: [Jan 30, 2024 8:00 AM (UTC)](https://zoom-lfx.platform.linuxfoundation.org/meeting/93514971498?password=907d67b6-3e04-42d3-afdc-52d288122c6f)

**Comment by jkowall** [comment]
The UTC time is a bit difficult for both of us as that is 3am for us in local time.

**Comment by nate-double-u** [comment]
Does the North America one work for you? They’ll be the same session, just run by different folks.

**Comment by jkowall** [comment]
Yes, that time works for us.

**Pull request merged**